chrome/background.js

```diff
@@ -459,7 +460,26 @@ function copyStringToClipboard(str) {
     document.execCommand("copy");
     // Remove temporary element
     document.body.removeChild(el);
-    }
+}
+
+/* 
+ * New function using chrome.scripting to inject the copyStringToClipboard into current active tab.
+ * This was the only "workaround" to having clipboards work in Manifest V3 since the Servcie Workers
+ * no longer have access to the DOM in V3, which breaks all the old functionality.
+ */
+function copyStringToClipboard(str) {
+    chrome.tabs.query({active: true, currentWindow: true}, function(tabs) {
+        var currTab = tabs[0];
+        if (currTab) 
+        {
+            chrome.scripting.executeScript({
+                target: {tabId: currTab.id},
+                func: injectCopyStringToClipboard,
+                args: [str],
+            });
+        }
+    });
+};
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/mitchmoser/sputnik/pullRequest?prid=8514944)
```suggestion
}
```